eng/Versioning.targets

```diff
@@ -4,7 +4,7 @@
     <_BuildNumber>$(OfficialBuildId)</_BuildNumber>
     <_BuildNumber Condition="'$(OfficialBuildId)' == ''">$([System.DateTime]::Now.ToString(yyyyMMdd)).1</_BuildNumber>
     <HasReleaseVersion>true</HasReleaseVersion>
-    <HasReleaseVersion Condition="$(Version.Contains('preview'))">false</HasReleaseVersion>
+    <HasReleaseVersion Condition="$(Version.Contains('preview')) or $(Version.Contains('beta'))">false</HasReleaseVersion>
```

Comment:
Should this check for alpha as well?

Reply:
No as that should never be committed into the project file.